test/features/step_definitions/sdk_steps.rb

```diff
@@ -54,13 +64,15 @@
     doc = 'confirm.national_identity_card.message'
   elsif document.include?('license') || document.include?('licence')
     doc = 'confirm.driving_licence.message'
+  else
+    face = 'confirm.face.standard.message'
   end
-  face = 'confirm.face.standard.message'
 
+  puts document
```

Comment:
This is probably a debug statement right?

Reply:
removed